manim/utils/bezier.py

```diff
@@ -216,46 +211,538 @@ def quadratic_bezier_remap(
 
     Returns
     -------
+    QuadraticBezierPoints_Array
         The new triplets for the quadratic bezier curves.
     """
-    difference = new_number_of_curves - len(triplets)
-    if difference <= 0:
-        return triplets
-    new_triplets = np.zeros((new_number_of_curves, 3, 3))
-    idx = 0
-    for triplet in triplets:
-        if difference > 0:
-            tmp_noc = int(np.ceil(difference / len(triplets))) + 1
-            tmp = subdivide_quadratic_bezier(triplet, tmp_noc).reshape(-1, 3, 3)
-            for i in range(tmp_noc):
-                new_triplets[idx + i] = tmp[i]
-            difference -= tmp_noc - 1
-            idx += tmp_noc
-        else:
-            new_triplets[idx] = triplet
-            idx += 1
-    return new_triplets
 
+    return bezier_remap(triplets, new_number_of_curves)
+
+
+def partial_bezier_points(points: BezierPoints, a: float, b: float) -> BezierPoints:
+    """Given an array of points which define a Bézier curve, and two numbers 0<=a<b<=1,
+    return an array of the same size, which describes the portion of the original Bézier
+    curve on the interval [a, b].
+
+    To understand the mathematics behind splitting curves, see split_bezier for an explanation.
+
+    To find the portion of a curve with t between a and b:
+    1. Split the curve at t = a and extract its 2nd subcurve.
+    2. We cannot evaluate the new subcurve at t = b because its range of values for t is different.
+       To find the correct value, we need to transform the interval [a, 1] into [0, 1]
+       by first subtracting a to get [0, 1-a] and then dividing by 1-a. Thus, our new
+       value must be t = (b - a) / (1 - a). Define u = (b-a) / (1-a).
+    3. Split the subcurve at t = u and extract its 1st subcurve.
```

Comment:
This should fix the numbered list, did not test locally though.

```suggestion
    To find the portion of a curve with t between a and b:

    #. Split the curve at t = a and extract its 2nd subcurve.
    #. We cannot evaluate the new subcurve at t = b because its range of values for t is different.
       To find the correct value, we need to transform the interval [a, 1] into [0, 1]
       by first subtracting a to get [0, 1-a] and then dividing by 1-a. Thus, our new
       value must be t = (b - a) / (1 - a). Define u = (b-a) / (1-a).
    #. Split the subcurve at t = u and extract its 1st subcurve.
```

Reply:
Fixed!
In the end I just needed to add an extra blank line before starting the list. When Sphinx encounters groups of consecutive lines (non-separated by blank lines), it tends to put them in the same paragraph.